moto/elbv2/models.py

```diff
@@ -833,6 +835,15 @@ def create_target_group(self, name, **kwargs):
         self.target_groups[target_group.arn] = target_group
         return target_group
 
+    def convert_and_validate_certificates(self, certificates):
+
+        # transform default certificate to conform with the rest of the code and XML templates
+        if certificates:
+            for cert in certificates:
```

Comment:
Let's simplify this to `for cert in certificates or []:` (so we can get rid of the if-statement)

Reply:
Done: 86d2773